From: Matthew Garrett <mjg@redhat.com>
Date: Mon, 18 Aug 2008 20:46:40 +0100
Subject: [misc] fix kernel builds on modern userland
Message-id: 20080818194640.GB1564@srcf.ucam.org
O-Subject: [RHEL5 patch] Fix kernel builds on modern userland
Bugzilla: 461540
RH-Acked-by: Jon Masters <jcm@redhat.com>
RH-Acked-by: Mark McLoughlin <markmc@redhat.com>

I seem to need the following diff to be able to do kernel builds under
rawhide. Any chance of it getting applied?

diff --git a/scripts/mod/sumversion.c b/scripts/mod/sumversion.c
index 8a28756..4f0a29c 100644
--- a/scripts/mod/sumversion.c
+++ b/scripts/mod/sumversion.c
@@ -7,6 +7,7 @@
 #include <ctype.h>
 #include <errno.h>
 #include <string.h>
+#include <limits.h>
 #include "modpost.h"
 
 /*
